.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,38 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
```

Comment:
As well as, or perhaps in preference of, screenshots, copy and paste of code being run is often more useful as those working can then use the commands and/or navigate it. This isn't possible with images.

```suggestion
**Output**

If applicable please include the output error, this can be a copy and paste of the output (preferable) or a screenshot.
```

Reply:
Good point!